presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -76,6 +76,12 @@ class SystemConfig : public ConfigBase {
   static constexpr std::string_view kPrestoVersion{"presto.version"};
   static constexpr std::string_view kHttpServerHttpPort{
       "http-server.http.port"};
+  // A port could be in TIME_WAIT status when it is closed for a while. Reusing
```

Comment:
It seems to help Presto native worker restart as well? Thanks!

Reply:
```
// This option allows a port closed in TIME_WAIT state to be reused immediately on worker restart. This property is mainly used by batch processing as the worker uses the same port on restart. For interactive query, the worker uses a dynamic port on restart.
```

Reply:
No I don't think it is going to reuse any port. I think it is going to just find an available port. This does not apply to that situation.